assets/js/helpers.ts

```diff
@@ -87,4 +87,22 @@ function formatTime(dtIso8601: string | null): string {
   return `${datePart} @ ${timePart}`;
 }
 
-export { choosePage, alertsForLine, formatTime };
+function defaultZoneLabel(z: Zone): string {
+  /* eslint consistent-return: "off", default-case: "off" -- typescript ensures exhaustive match */
```

Comment:
Are we already using a TypeScript profile for ESLint? Seems like it should include this, unless the exhaustive match is an optional feature.

Reply:
I think we are. Frank switched us from tslint -> eslint, and our config seems to follow [the docs](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/README.md).

The `consistent-return` rule isn't *just* about types, strictly speaking. Some people don't seem to like the idea that a function can both early return and also just finish evaluating without returning anything, even if the type is `| undefined`. Similarly, `default-case` I guess is a style that some people like where a switch always has a `default`, even if it's empty.

But, personally, I don't think those provide much value on top of what TypeScript already gets us, and the `tsc` check in `npm run check` will highlight if a switch doesn't exhaustively handle cases, or if we have a code branch that doesn't return anything when it should.

So I suppose I'll remove the comment here and just disable those rules in the project instead.